browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -182,6 +182,7 @@ function _extractText(selectCommandData) {
 JSDialog.combobox = function (parentContainer, data, builder) {
 	var container = L.DomUtil.create('div', 'ui-combobox ' + builder.options.cssClass, parentContainer);
 	container.id = data.id;
+	container.role = 'listbox';
```

Comment:
looks good, but we should in the future distinguish when it is a "listbox" and "comobox"